src/components/dialogs/network-modifications/by-filter-deletion/by-filter-deletion-form.tsx

```diff
@@ -44,9 +44,17 @@ const ByFilterDeletionForm = () => {
             EQUIPMENT_TYPES.BUSBAR_SECTION,
             EQUIPMENT_TYPES.TIE_LINE,
         ]);
-        return Object.values(EQUIPMENT_TYPES).filter(
-            (equipmentType) => !equipmentTypesToExclude.has(equipmentType)
-        );
+        return Object.values(EQUIPMENT_TYPES)
+            .filter(
+                (equipmentType) => !equipmentTypesToExclude.has(equipmentType)
+            )
+            .map((element) => {
+                if (element === EQUIPMENT_TYPES.HVDC_LINE) {
+                    return 'Hvdc';
+                } else {
+                    return element;
+                }
+            });
```

Comment:
```suggestion
           .map((equipmentType) =>
                equipmentType === EQUIPMENT_TYPES.HVDC_LINE
                    ? 'Hvdc'
                    : equipmentType
            );
```
Ternary operator is not important, do as you wish. But I think it is better to use the more precise 'equipmentType' than the generic 'element'.

Reply:
Yes I changed it